src/oracles.jl

```diff
@@ -303,3 +303,61 @@ function compute_extreme_point(
     end
     return storage
 end
+
+"""
+"""
+struct ChasingGradientLMO{LMO<:LinearMinimizationOracle,T} <: LinearMinimizationOracle
+    inner::LMO
+    max_rounds_number::Int
+    improv_tol::T
+end
+
+"""
+"""
+function compute_extreme_point(lmo::ChasingGradientLMO, direction; x, kwargs...)
+    d = 0 * direction
+    Λ = 0
+    #@debug x
+    #@debug direction
+    if norm(direction) == 0
```

Comment:
```suggestion
    if norm(direction) <= eps(eltype(direction))
```

this makes your check robust to numerical errors. Same applies below